archunit/src/main/java/com/tngtech/archunit/core/importer/ImportOption.java

```diff
@@ -137,4 +149,16 @@ public boolean includes(Location location) {
             return !location.isArchive();
         }
     }
+
+    /**
+     * The {@code maven-compiler-plugin 3.10.0} fixed a bug that made {@code package-info.class}
```

Comment:
I don't think we need this context information inside of the Javadoc? This sounds to me like the reason you stumbled over it, but `package-info.class` has been there long before, I have already used it with ArchUnit, Gradle and IntelliJ long ago :man_shrugging: This is for me context information that could go into the commit message or PR message, but I don't think it's relevant for users who want to read the Javadoc of this `ImportOption`. In the Javadoc I would rather expect to see an explanation what `package-info` actually is (maybe with a link to the JLS) and that this `ImportOption` filters it out :thinking:

Reply:
good point.

Reply:
good point.
cleaned it up in https://github.com/TNG/ArchUnit/pull/794/commits/90c126f6acba68e0602e316c5d576001ee5f20ce